doc/model/varia/functions/userSelectionAction.xml

```diff
@@ -21,6 +21,12 @@
                 An array with the datums that will be selected by the current operation.
             </c:documentation>
         </c:argument>
+        <c:argument name="selectingGroup" type="list(pvc.data.Data)">
+            <c:documentation>
+                An array with the group datums if the current operation will only select a single group, otherwise no
```

Comment:
The type should be `pvc.data.Data`, not a list of.
The description could be: A data group, if the current operation...